tensorflow/contrib/gan/python/estimator/python/gan_estimator_impl.py

```diff
@@ -214,12 +214,18 @@ def _get_gan_model(
 def _get_estimator_spec(
     mode, gan_model, generator_loss_fn, discriminator_loss_fn,
     get_eval_metric_ops_fn, generator_optimizer, discriminator_optimizer,
-    get_hooks_fn=None):
+    get_hooks_fn=None, use_loss_summaries=True):
   """Get the EstimatorSpec for the current mode."""
   if mode == model_fn_lib.ModeKeys.PREDICT:
     estimator_spec = model_fn_lib.EstimatorSpec(
         mode=mode, predictions=gan_model.generated_data)
   else:
+    if 'add_summaries' in inspect.getargspec(generator_loss_fn).args:
```

Comment:
The previous code didn't have this check. Furthermore, the losses test checks that this arg is present: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/gan/python/losses/python/losses_impl_test.py#L158

How would you feel about removing this argument?

Reply:
I'm happy to remove this check.
I wasn't sure since the estimator tests failed otherwise due to the missing argument in the dummy loss used there.

Reply:
Ah good point. Would you mind adding that argument to the dummy loss function? Sorry for that, and great catch!